### PR TITLE
Copyright in files

### PR DESCRIPTION
**Pull request by Liaojinghui**

### CODE REVIEW

**Comment by vncoelho** [comment]
Looks, good, I think @erikzhang should double check it.

**Review by shargon — DISMISSED** [review]
Look good, but maybe redundant with the LICENSE file

**Comment by Liaojinghui** [comment]
> Look good, but maybe redundant with the LICENSE file

On the contrary, the LICENSE file is far from enough for a public open-source project that welcomes contributions from the community and allows anyone to use its code. To the best of my knowledge, all open source projects as famous as neo have in-file copyright and license declaration.

**Review by superboyiii — DISMISSED** [review]

**Comment by superboyiii** [comment]
@erikzhang Merge?

File: Neo.ConsoleService/CommandQuoteToken.cs [hunk 1]
```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2016-2021 NEO GLOBAL DEVELOPMENT.
```
Comment (erikzhang) [inline Neo.ConsoleService/CommandQuoteToken.cs:1]:
```suggestion
// Copyright (C) 2016-2021 The Neo Project.
```

**Review by erikzhang — CHANGES_REQUESTED** [review]

File: Neo.ConsoleService/CommandQuoteToken.cs [hunk 2]
```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2016-2021 The Neo Project.
+// 
+// The neo-consolservice is free software distributed under the MIT software license,
```
Comment (erikzhang) [inline Neo.ConsoleService/CommandQuoteToken.cs:3]:
```suggestion
// The Neo.ConsoleService is free software distributed under the MIT software license, 
```

File: neo-cli/CLI/ConsolePercent.cs [hunk 3]
```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2016-2021 The Neo Project.
+// 
+// The Neo.CLI is free software distributed under the MIT software
```
Comment (erikzhang) [inline neo-cli/CLI/ConsolePercent.cs:3]:
```suggestion
// The neo-cli is free software distributed under the MIT software 
```

It should be the same as the project name.

**Review by erikzhang — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Pull request merged**